components/changelog/buildComponent.tsx

```diff
@@ -27,20 +27,19 @@ const BuildComponent = (props: Build) => {
             <li className={'py-3 sm:py-4'}>
                 <div className={'flex items-center space-x-4'}>
                     <div className={'shrink-0'}>
-
+                        
                     </div>
                     <div className={'min-w-fit flex-1'}>
                         <p className={'text-center text-gray-500'}>This build has no registered changes :(</p>
                     </div>
                 </div>
             </li>
     }
-
     return (
         <div className={'max-w-xl min-w-sm'}>
             <Card>
                 <div className={'mb-4 flex justify-between'}>
-                    <h5 className="text-xl font-bold leading-none text-white">
+                    <h5 className="text-xl font-bold leading-none text-yellow-300">
```

Comment:
Making this colored would make things not look consistent or pleasing, as there would be no rhythm to it nor a general pattern. I'd say keep it white, as yellow is also a color that doesn't fit with the background colors.

Reply:
The point is to highlight important information with a warm colour and a neutral connotation, such as yellow or orange or brown.

For example, the build number should not be lost in the text. 
I based this modification on the elements previously provided to gilles on discord. I just took some time to understand the documentation.

Reply:
I also think the build number should be white. 

There is another thing we have to solve, but we can do that in another PR. I think you can lose the build number text because it is way to close to the previos build information, so we should separate that.

We can discuss more options to highlight the build number but for now, please leave it white.